app/views/issues/index.html.erb

```diff
@@ -60,6 +61,13 @@
           <td class="text-nowrap"><%= link_to t(".reports_count", :count => issue.reports_count), issue %></td>
           <td><%= link_to reportable_title(issue.reportable), reportable_url(issue.reportable) %></td>
           <td><%= link_to issue.reported_user.display_name, issue.reported_user if issue.reported_user %></td>
+          <td>
+            <% unique_reporters = issue.reports.uniq { |report| report.user.id } %>
```

Comment:
This could probably be better done as `issue.reports.preload(:user).map(&:user).uniq` which would also get a list of users instead of a list of reports.

Reply:
To avoid Ruby on Rails logic code in the HTML, `preload` logic was moved in the controller.